controller/schema.mustache

```diff
@@ -1,5 +1,7 @@
 {
   "type": "object",
+  "name": "{{name}}",
```

Comment:
We're not off spec, we're just outside spec. These extra properties are a redbeard-specific convention. We're still totally spec compliant.

If we can think of a way to do this more inside the schema spec then brilliant! I can't think of one, though.

There's a need for this over in redbeard-admin at least.

https://github.com/Prismatik/redbeard-admin/blob/master/index.js#L25